provider/internal/device/ftd/resource.go

```diff
@@ -109,6 +110,14 @@ func (r *Resource) Schema(ctx context.Context, req resource.SchemaRequest, resp
 					validators.ValueStringsAtLeast(stringvalidator.OneOf(string(license.Base))),
 				},
 			},
+			"tags": schema.ListAttribute{
```

Comment:
I was just reading the CDO documentation: we call tags labels in the documentation. So let's call them labels here in the provider (let's continue to call them tags in the client, because that's what they are internally).

Could you also please change the markdown description to: "Set a list of labels to identify the device as part of a group. Refer to the [CDO documentation](https://docs.defenseorchestrator.com/t-applying-labels-to-devices-and-objects.html#!c-labels-and-filtering.html) for details on how labels are used in CDO."

Reply:
done